libvirt/resource_libvirt_network.go

```diff
@@ -115,6 +116,52 @@ func resourceLibvirtNetwork() *schema.Resource {
 								},
 							},
 						},
+						"srvs": {
+							Type:     schema.TypeList,
+							Optional: true,
+							ForceNew: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"name": {
+										Type: schema.TypeString,
+										// This should be required, but Terraform does validation too early
+										// and therefore doesn't recognize that this is set when assigning from
+										// a rendered dns_host template.
+										Optional: true,
+										ForceNew: true,
+									},
+									"proto": {
+										Type: schema.TypeString,
+										// This should be required, but Terraform does validation too early
```

Comment:
Referring to this

Reply:
This is the same issue we faced when adding `dns.hosts` in https://github.com/dmacvicar/terraform-provider-libvirt/pull/382
similar comment on https://github.com/dmacvicar/terraform-provider-libvirt/pull/460/files/de77bbb1d412cac90118176391ba9d349d1a38cc#diff-042e2374df248e4277a3506b065eb28aR174

i'm not 100% on terraform but, it looks like terraform complains in plan phase when the list from dns srv template has not been computed.

Reply:
Yop, i think we might have an issue just for remember it and fix it later. I will havw a look once i have terminal :)

Reply:
Yop, i think we might have an issue just for remember it and fix it later. I will havw a look once i have terminal :)

Reply:
:+1: Thanks!